spec/bundler/dsl_spec.rb

```diff
@@ -180,4 +180,14 @@
         to raise_error(Bundler::GemfileError, /Gemfile syntax error/)
     end
   end
+
+  if RUBY_VERSION > "1.9"
+    describe "Runtime errors" do
+      it "will raise a Bundler::GemfileError" do
+        gemfile "s = 'foo'.freeze; s.strip!"
```

Comment:
I didnt want to force raise a RuntimeError, in case that if ruby changes what `freeze` raises we would catch that on this regression test.